src/shared/analytics/analytics.background.ts

```diff
@@ -81,66 +87,104 @@ function trackAppEvents({ account }: { account: Account }) {
       feeValueCommon,
       addressAction,
       quote,
+      clientScope,
     }) => {
       const initiatorURL = new URL(initiator);
       const { origin, pathname } = initiatorURL;
+      const isInternalOrigin = globalThis.location.origin === origin;
+      const initiatorName = isInternalOrigin ? 'Extension' : 'External Dapp';
       const networks = await networksStore.load();
       const chainId = ethers.utils.hexValue(transaction.chainId);
       const chain = networks.getChainById(chainId)?.toString() || chainId;
+      const addressActionAnalytics = addressActionToAnalytics({
+        addressAction,
+        quote,
+      });
       const params = createBaseParams({
         request_name: 'signed_transaction',
         screen_name: origin === initiator ? 'Transaction Request' : pathname,
         wallet_address: transaction.from,
         wallet_provider: getProvider(transaction.from),
-        context:
-          globalThis.location.origin === origin ? 'Extension' : 'External Dapp',
-        type: 'sign',
-        dapp_domain: globalThis.location.origin === origin ? null : origin,
+        /* @deprecated */
+        context: initiatorName,
+        /* @deprecated */
+        type: 'Sign',
+        client_scope: clientScope ?? initiatorName,
+        action_type: addressActionAnalytics?.action_type ?? 'Execute',
+        dapp_domain: isInternalOrigin ? null : origin,
         chain,
         gas: transaction.gasLimit.toString(),
         hash: transaction.hash,
         asset_amount_sent: [], // TODO
         gas_price: null, // TODO
         network_fee: null, // TODO
         network_fee_value: feeValueCommon,
-        ...addressActionToAnalytics({ addressAction, quote }),
+        contract_type: quote?.contract_metadata?.name ?? null,
+        ...addressActionAnalytics,
       });
       sendToMetabase('signed_transaction', params);
+      const mixpanelParams = omit(params, [
+        'request_name',
+        'hash',
+        'wallet_address',
+      ]);
+      mixPanelTrack(account, 'Transaction: Signed Transaction', mixpanelParams);
     }
   );
 
   function handleSign({
     type,
     initiator,
     address,
+    clientScope,
   }: {
     type: 'typedDataSigned' | 'messageSigned';
     initiator: string;
     address: string;
+    clientScope: string | null;
   }) {
-    if (initiator === INTERNAL_ORIGIN) {
+    if (!clientScope && initiator === INTERNAL_ORIGIN) {
```

Comment:
Ok, now it is more clear why we didn't add clientScope for sign to promote DNA, but I will leave the comment anyway just in case. Not sure that not having clientScope is the best way to prevent sending the event. May be it is better to provide something more obvious?